static/js/compose_closed_ui.js

```diff
@@ -62,7 +63,10 @@ function update_conversation_button(btn_text, title) {
 function update_buttons(text_stream) {
     const title_stream = text_stream + " (c)";
     const text_conversation = $t({defaultMessage: "New private message"});
-    const title_conversation = text_conversation + " (x)";
+    let title_conversation = text_conversation + " (x)";
+    if (page_params.realm_private_message_policy === 2) {
```

Comment:
can you use the symbolic name instead of the hardcoded value?  for example:
```
./popovers.js:231:            page_params.realm_private_message_policy !==
./popovers.js-232-                settings_config.private_message_policy_values.disabled.code,
```

Reply:
Yeah sure, I'll look into it